rhacm/policies/CP-Contingency_Planning/component.yaml

```diff
@@ -0,0 +1,702 @@
+##
+## BEGINNING OF:
+## CONTINGENCY PLANNING
+##
+## Reminder of "implementation_status" codes:
+##  - implementation_status: unsatisfied
+##  - implementation_status: unknown
+##  - implementation_status: planned
+##  - implementation_status: partial
+##  - implementation_status: complete
+##  - implementation_status: none
+
+- control_key: CP-1
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - key: a
+      text: |
```

Comment:
change `|` to `|-`

Reply:
Thought I fixed it... guess not. Should be fixed now.